api/client/samples/crop_w_series/soybean_w_drought_index.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>It will help to mention that the <code>EVAL_DATE</code>  variable is a constant users can change to visualize the weighted Gro Drought Index at different dates.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Done</p>